Makefile

```diff
@@ -153,6 +153,9 @@ stop-db: check-db_type
 .PHONY: build
 build: init init-db test-certs scheduler autoscaler
 
+.PHONY: just-build
+just-build: init scheduler autoscaler
```

Comment:
Feel a bit weird: `build` depends on the db, which seems a bit wrong, but I did not want to break `build` users.

Reply:
true ... might need a lil tweek.

Reply:
hmmm I think having one target build and not just build is better. 
Lets just make it work.

Reply:
Done